inc/commonitilobject.class.php

```diff
@@ -7038,21 +7042,25 @@ function showTimeline($rand) {
 
                echo "<span class='h_user_name'>";
                $userdata = getUserName($item_i['users_id'], 2);
```

Comment:
Unless I am wrong, you could save some requests by moving this in the `else` block.

Reply:
You are right, done in b3416b7.